Tests/GameObjectExtensionsTests.cs

```diff
@@ -31,11 +31,11 @@ public void OneTimeSetup()
 			}
 
 			prefabBehaviour = new GameObject(PREFAB_WITHOUT_ARGS_RESOURCE_PATH).AddComponent<TestHierarchyBehaviour>();
-			PrefabUtility.CreatePrefab("Assets/Resources/" + PREFAB_WITHOUT_ARGS_RESOURCE_PATH + ".prefab", prefabBehaviour.gameObject);
+			PrefabUtility.SaveAsPrefabAsset(prefabBehaviour.gameObject, "Assets/Resources/" + PREFAB_WITHOUT_ARGS_RESOURCE_PATH + ".prefab");
```

Comment:
You could use string interpolation here
`$"Assets/Resources/{PREFAB_WITHOUT_ARGS_RESOURCE_PATH}.prefab"`

Reply:
Yeah would be good.